test/e2e/common/configmap_volume.go

```diff
@@ -33,18 +33,18 @@ var _ = Describe("[sig-storage] ConfigMap", func() {
 	f := framework.NewDefaultFramework("configmap")
 
 	/*
-		    Testname: configmap-nomap-simple
-		    Description: Make sure config map without mappings works by mounting it
-			to a volume with a custom path (mapping) on the pod with no other settings.
+		Release : v1.9
+		Testname: ConfigMap without mapping
+		Description: Create a configMap, create a Pod that mounts a volume and populates the volume with data stored in the configMap. The config map that is created MUST be accessible to read from the newly created Pod using the volume mount. The data content of the file MUST be readable and verified and file modes MUST default  to 0x644.
 	*/
 	framework.ConformanceIt("should be consumable from pods in volume [NodeConformance]", func() {
 		doConfigMapE2EWithoutMappings(f, 0, 0, nil)
 	})
 
 	/*
-		    Testname: configmap-nomap-default-mode
-		    Description: Make sure config map without mappings works by mounting it
-			to a volume with a custom path (mapping) on the pod with defaultMode set
+		Release : v1.9
+		Testname: ConfigMap without mapping, volume mode set
+		Description: Create a configMap, create a Pod that mounts a volume and populates the volume with data stored in the configMap. The config map that is created MUST be accessible to read from the newly created Pod using the volume mount. The data content of the file MUST be readable and verified and file modes MUST be set to the custom value of ‘0x400’
```

Comment:
I don't know these two tests well enough, but what does "without mappings" mean? The last sentence looks like a check for file mode (volume mode) but there should be a sentence about setting the volume mode before checking it, right?

Reply:
I have added description saying file mode changed to custom value. About the test names stating  'with mapping' (items to file paths) or 'without mapping', I am also looking for suggestions.